token.go

```diff
@@ -4,24 +4,17 @@ import (
 	"encoding/base64"
 	"encoding/json"
 	"strings"
-	"sync/atomic"
 	"time"
 )
 
-const (
-	DisablePadding uint64 = 0 // Utilizes RawURLEncoding
-	AllowPadding   uint64 = 1 // Utilizes URLEncoding
-
-)
-
-var decodePaddingAllowed uint64
+var decodePaddingAllowed bool
```

Comment:
For consistency with existing variables you can just export it and get rid of the setter function

Reply:
Done!